doc/api/test.md

```diff
@@ -851,17 +850,17 @@ Shorthand for marking a suite as `only`, same as
 
 ## `it([name][, options][, fn])`
 
-* `name` {string} The name of the test, which is displayed when reporting test
-  results. **Default:** The `name` property of `fn`, or `'<anonymous>'` if `fn`
-  does not have a name.
-* `options` {Object} Configuration options for the suite.
-  supports the same options as `test([name][, options][, fn])`.
-* `fn` {Function|AsyncFunction} The function under test.
-  If the test uses callbacks, the callback function is passed as an argument.
-  **Default:** A no-op function.
-* Returns: `undefined`.
+<!-- YAML
+added: v18.6.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/46889
+    description: it is now a shorthand for test and behaves the same way.
```

Comment:
Is it actually a shorthand and we have `test.skip` and `test.only` now, or is it only when calling them that gives similar result?
```suggestion
    description: Calling `it()` is now equivalent to calling `test()`.
```